editor.planx.uk/src/ui/editor/ModalFooter.tsx

```diff
@@ -0,0 +1,45 @@
+import { BaseNodeData } from "@planx/components/shared";
+import { InternalNotes, MoreInformation } from "@planx/components/ui"
+import { useFormik } from "formik";
+import React from "react";
+
+import { ComponentTagSelect } from "./ComponentTagSelect"
+
+interface Props<T extends BaseNodeData> {
+  formik: ReturnType<typeof useFormik<T>>
+  showMoreInformation?: boolean
+  showInternalNotes?: boolean
+  showTags?: boolean
+}
+
+export const ModalFooter = <T extends BaseNodeData>({
+  formik,
+  showMoreInformation = true,
+  showInternalNotes = true,
+  showTags = true,
+}: Props<T>) => (
+  <>
+    {showMoreInformation &&
+      <MoreInformation
+        changeField={formik.handleChange}
+        definitionImg={formik.values.definitionImg}
+        howMeasured={formik.values.howMeasured}
+        policyRef={formik.values.policyRef}
+        info={formik.values.info}
+      />
+    }
+    {showInternalNotes &&
+      <InternalNotes
+      name="notes"
+      onChange={formik.handleChange}
+      value={formik.values.notes}
+      />
+    }
+    { showTags &&
```

Comment:
nit: unnecessary leading space here inconsistent with preceding conditions (surprised precommit hook didn't catch!)

Reply:
Thanks, fixed in [cda35f8](https://github.com/theopensystemslab/planx-new/pull/3762/commits/cda35f88d404a3812193b79fd1edeb7c3e1e36d1)